src/Blocks/ChurchBellPartBlock.cs

```diff
@@ -0,0 +1,50 @@
+﻿using RPVoiceChat.BlockEntities;
+using Vintagestory.API.Common;
+using Vintagestory.API.MathTools;
+
+namespace RPVoiceChat.Blocks
+{
+    public class ChurchBellPartBlock : Block
```

Comment:
This class is a duplication of ChurchBellLayerBlock, it looks like church bell layers shouldn't exist as an separate entity - they share most if not all of the logic with parts and any unique logic can be attributed to parts in favor of removing layers